docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# ParaTime Observer Node
+
+:::info
+
+These instructions are for setting up a _ParaTime observer_ node, which observes ParaTime activity and can issue
+queries that can access confidential data. If you just want to run a _ParaTime client_ node, see the
+[instructions for running a ParaTime client node](paratime-client-node.mdx). If you want to run a _ParaTime_ node
+instead, see the [instructions for running a ParaTime node](paratime-node.mdx). Similarly, if you want to run a
+_validator_ or a _non-validator_ node instead, see the [instructions for running a validator node](validator-node.mdx)
+or [instructions for running a non-validator node](non-validator-node.mdx).
+
+:::
+
+:::tip
+
+The observer node role allows any ParaTime client node with appropriate TEE support to run confidential transactions
+and view calls on Sapphire ParaTime. Observer nodes are client nodes that are registered on chain so that their
+eligibility can be enforced by an on-chain policy (e.g. key manager committees can grant them permissions).
+
+:::
+
+This guide will cover setting up your ParaTime observer node for the Oasis Network. Observer nodes are ParaTime
+client nodes that support confidential queries without being elected into the compute committee. This guide assumes
+some basic knowledge on the use of command line tools.
+
+## Prerequisites
+
+Before following this guide, make sure you've followed the [Prerequisites](prerequisites),
+[Run a Non-validator Node](non-validator-node.mdx), and [Run a ParaTime client Node](paratime-client-node.mdx)
+sections and have a working ParaTime client node with Intel SGX support.
+
+### Stake Requirements
+
+To be able to register as a ParaTime observer node on the Oasis Network, you need to have enough tokens staked in your
+entity's escrow account.
+
+Current minimum staking requirements for a specific ParaTime are listed on the [Stake Requirements] page. Should you
+want to check the staking requirements for other node roles and registered ParaTimes manually, use the Oasis CLI tools
+as described in [Common Staking Info].
+
+Finally, to stake the tokens, use our [Oasis CLI tools]. If everything was set up correctly, you should see something
+like below when running [`oasis account show`] command for your entity's account (this is an example for Testnet):
+
+![code shell](../../../external/cli/examples/account/show-delegations.in.static)
+
+![code](../../../external/cli/examples/account/show-delegations.out.static)
+
+:::caution
+
+The stake requirements may differ from ParaTime to ParaTime and are subject to change in the future.
+
+:::
+
+:::info
+
+For example, if you want to register an observer node for Testnet/Mainnet, you currently need to have at least
```

Comment:
Move these current requirements with the notice that they are subject to change in the future above.

Reply:
Moved up.